test/e2e/tests/add-account.spec.js

```diff
@@ -36,4 +44,216 @@ describe('Add account', function () {
       },
     );
   });
+
+  it('should add the same account addresses if added and then the same mnemonic is imported', async function () {
+    await withFixtures(
+      {
+        fixtures: 'onboarding',
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
```

Comment:
It might be useful to extract onboarding part into separate method that can be reused in other account / onboarding tests also.

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/13668/commits/578c813a526cd61d974e706cc3839bcee7ef1c12